pyxem/utils/indexation_utils.py

```diff
@@ -603,16 +555,42 @@ def _match_polar_to_library_cpu(
     N is the number of templates and R the number of spots in the template
     with the maximum number of spots
     """
-    correlations = _correlate_polar_to_library_cpu(
-        polar_image, r_templates, theta_templates, intensities_templates
-    )
-    correlations_mirror = _correlate_polar_to_library_cpu(
-        polar_image,
-        r_templates,
-        (polar_image.shape[0] - theta_templates) % polar_image.shape[0],
-        intensities_templates,
+    N = r_templates.shape[0]
+    D = r_templates.shape[1]
```

Comment:
Should change this to `R` to be in line with docstring, or change docstring.

Reply:
done